doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -71,6 +71,19 @@ Peripheral samples
   * The CLI command ``fem tx_power_control <tx_power_control>`` replaces ``fem tx_gain <tx_gain>`` .
     This change applies to the sample built with the :ref:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC <CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC>` set to ``n``.
 
+
+Matter
+------
+
+  * With the inheritance of Zephyr's :ref:`zephyr:sysbuild` in the |NCS|, some changes are provided to the Matter samples and applications:
+
+    * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD` kconfig option is deprecated and you need to use the :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA` Kconfig option instead to enable or disable creating the factory data set during building a Matter sample.
+      To enable factory data support on your device, you still need to set the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA` to ``y``.
+    * Factory data output files are located now in the ``<application_name>/zephyr/`` directory within the sample build directory.
```

Comment:
*are now located in

Would also not put sample build directory, just build directory, because it's the same for samples and user applications